test/integration/offinstall_int_test.go

```diff
@@ -171,6 +171,77 @@ func (suite *OffInstallIntegrationTestSuite) TestInstallNoPermission() {
 	tp.ExpectExitCode(1)
 }
 
+func (suite *OffInstallIntegrationTestSuite) TestInstallTwice() {
+	suite.OnlyRunForTags(tagsuite.OffInstall)
+
+	// Clean up env after test (windows only for now)
```

Comment:
Why is this Windows only? Since this test runs on all platforms, is there a risk the environment will not be cleaned up for macOS and Linux? Will there be any ill effects?

Reply:
Fair point. This was copied from another test that already existed. I've updated all of the tests to clean up after themselves

Reply:
Okay great. Please update the comment then.